nginx.conf

```diff
@@ -0,0 +1,21 @@
+events {
+}
+http {
+  # This path may be different on your local computer depending on how you
+  # installed nginx.
+  include /usr/local/etc/nginx/mime.types;
+
+  server {
+    listen 8080;
```

Comment:
would recommend a more esoteric port (even something like 8989) in case someone is using this already locally.

Reply:
I'm in favor of this. Ran into a conflict earlier today.

Reply:
Sounds good. I'll switch it to 8989 in https://github.com/nafundi/super-adventure/pull/24.